### DISCUSSION TITLE
Wireguard doesn't work

### DISCUSSION THREAD

**Discussion by massigent** [linuxmint/discussions#505] [Development]
With the last update wireguard doesn't work.
I used timeshift to restore a backup of 10 days ago and it worked.
I install the updates and now it doesn't work.
Can you fix this problem?
thank you

**Comment by massigent** [comment]
It still doesn't work.

**Comment by Fantu** [comment]
Hi, you don't give any details about your problem, so it's impossible to help you.
For example, you have the issue on a device with wireguard as client? If yes you try to check if other devices can connect to same wireguard server, to exclude possible issue on server side?
Have you tried to look log to see what error have wireguard?

**Comment by massigent** [comment]
I solve the problem. 
it was the kernel. 
I was on 6.2 kernel and now I go back to 5.15. 
it works correctly. 
I had no problems.
